lib/rules/no-callback-literal.js

```diff
@@ -62,7 +62,8 @@ function couldBeError(node) {
         case "TaggedTemplateExpression":
         case "YieldExpression":
             return true // possibly an error object.
-
+        case "Literal":
+            return node.value == null
```

Comment:
according its docs, the arg can be null/undefined, so I changed `===` to `==`, to allow cases like `cb(undefined)`